packages/react/src/components/TimePicker/TimePicker.tsx

```diff
@@ -52,8 +52,8 @@ export const TimePicker: React.FunctionComponent<ITimePickerProps> = ({
 
   const optionsCount = getDropdownOptionsCount(increments, timeRange);
 
-  const initialValue = useConst(defaultValue || new Date());
-  const baseDate: Date = React.useMemo(() => generateBaseDate(increments, timeRange, initialValue), [
+  const [initialValue] = useControllableValue(defaultValue, new Date());
```

Comment:
I don't think this is the correct change. The [first argument to useControllableValue](https://github.com/microsoft/fluentui/blob/master/packages/react-hooks/src/useControllableValue.ts#L41) is meant to be the [controlled value](https://reactjs.org/docs/forms.html#controlled-components). [defaultValue is React's way of handling uncontrolled values](https://reactjs.org/docs/uncontrolled-components.html#default-values).

I think this change needs to update `TimePicker` to support both controlled and uncontrolled usage. However, I'm not 100% certain how that should be handled as I believe `TimePicker` is used more like a `ComboBox` so adding a controlled `value` may not make sense.

@CheerfulSatchel, do you have any ideas on how to proceed here?

Reply:
But defaultValue will be passed as props. If it is used with datepicker then default value will hold the value selected in datepicker.

I think in other controls also this scenario is handled in same way.

Reply:
@spmonahan @CheerfulSatchel  have we reached on any consensus for this issue?